frontend/storage/reducers/gameVisibilityReducer.ts

```diff
@@ -1,9 +1,10 @@
 import { AppStore, SET_GAME_VISIBILITY, AppActions } from '../common';
 
 export function gameVisibilityReducer(state: AppStore, action: AppActions): AppStore {
+	console.log(action);
 	switch (action.type) {
 		case SET_GAME_VISIBILITY: {
-			if (state.gamesVisibility.get(action.gameId)) {
+			if (state.gamesVisibility.has(action.gameId)) {
```

Comment:
Actually I think that this whole condition is not needed - when a new game is loaded/added you want to be able to set its visibility.